speedscale/README.md

```diff
@@ -0,0 +1,58 @@
+# Agent Check: Speedscale
+
+## Overview
+
+This integration publishes traffic replay results from [Speedscale][1] into Datadog. This lets you combine your observability data from Datadog with the results of a particular replay to drill-down into the root cause of poor performance. Find and troubleshoot potential performance issues before they show up in production with a combination of Speedscale and Datadog.
+
+## Setup
+
+### Configuration
+
+In order to utilize the integration you need to capture the following from Datadog:
+
+* [API Key][2] - An API key is required by the Datadog Agent to submit metrics and events to Datadog
+
+A best practice is to save this into an environment variable. Most likely you will store this environment variable in your continuous integration system, but when doing a one-off test you can access in your terminal like so:
```

Comment:
```suggestion
1. To use this integration you need a Datadog [API Key][2] so that the Datadog Agent can submit metrics and events to Datadog.

   A best practice is to save this into an environment variable. Most likely you will store this environment variable in your continuous integration system, but when doing a one-off test you can access in your terminal like so:
```